src/components/form-builder/designer/designer.tsx

```
@@ -253,22 +253,22 @@ function DesignerElementWrapper(props: { element: FormElementInstance }) {
-                    <div className="absolute left-1/2 top-1/2 -translate-x-1/2 -translate-y-1/2 animate-pulse">
-                        <p className="text-sm text-muted-foreground">
+                    <div className="absolute top-1/2 left-1/2 -translate-x-1/2 -translate-y-1/2 animate-pulse">
+                        <p className="text-muted-foreground text-sm">
                             Click for properties or drag to move
                         </p>
                     </div>
                 </div>
             )}
             {top_half.isOver && (
-                <div className="absolute top-0 h-[7px] w-full rounded-b-none rounded-t-xl bg-primary"></div>
+                <div className="bg-primary absolute top-0 h-[7px] w-full rounded-t-xl rounded-b-none"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)